js/transForms.js

```diff
@@ -0,0 +1,30 @@
+// IE doesn't support locale number formats
+const decimalMark = (element, value) => [
+	element.type == 'number',
+	Number(value.replace(',', '.')),
+]
+
+/* We are simulating a recruitment,
+hence requesting salaries with the new size of the entreprise */
+const incrementEffectif = (element, value) => [
+	element.name == 'effectif_entreprise',
+	value + 1,
+]
+
+const dureeLegaleMensuelle = 151.66,
+	dureeLegaleHebdomadaire = 35
```

Comment:
> I think it would be better to use let in most cases and const for true constants

No! [Please favour `const`](https://mathiasbynens.be/notes/es6-const). `let` then gives a hint that you intend to rebind the variable.
`const` cannot guarantee immutability anyway, so you'll always have the need for uppercasing actual constants.